### PR TITLE
Fix Style/EmptyLineAfterMagicComment message

### PR DESCRIPTION
**Pull request by wafcio**
The current message is little unclear. It doesn't mean that user must add a new line.

-----------------

Before submitting the PR make sure the following are checked:

* [ ] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [ ] Used the same coding conventions as the rest of the project.
* [ ] Feature branch is up-to-date with `master` (if not - rebase it).
* [ ] Squashed related commits together.
* [ ] Added tests.
* [ ] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [ ] All tests are passing.
* [ ] The new code doesn't generate RuboCop offenses.
* [ ] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [ ] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: lib/rubocop/cop/style/empty_line_after_magic_comment.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ module Style
       #     # Some code
       #   end
       class EmptyLineAfterMagicComment < Cop
-        MSG = 'Add a space after magic comments.'.freeze
+        MSG = 'Add a new line after magic comments.'.freeze
```
Comment (bbatsov) [inline lib/rubocop/cop/style/empty_line_after_magic_comment.rb]: new -> empty/blank

**Comment by wafcio** [comment]
@bbatsov what do you think about changing the description in YML file: https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml#L569? There is new line also.

**Comment by bbatsov** [comment]
@wafcio Yeah, it should be changed there as well. `newline` is not the same as blank line. :-)

**Comment by wafcio** [comment]
I decided to use an empty line as it appears in other rules with EmptyLine name.

**Pull request merged**